### DISCUSSION TITLE
Not sure if this is bug or not, but is there a way to make the background transparent?

### DISCUSSION THREAD

**Discussion by samuel-barkley** [st3w/neo#24] [Q&A]
The original cmatrix has a transparent background which works well with transparent terminals. I much prefer neo's functionality to cmatrix' but neo has a black (I think) background which doesn't allow for a transparent terminal. Is there a way to achieve this? Thanks for reading. 🙂

cmatrix: 
![image](https://user-images.githubusercontent.com/60850711/187053647-86a2df34-f731-4994-90cd-93033d59c90d.png)

neo:
![image](https://user-images.githubusercontent.com/60850711/187053663-3999104d-303f-4698-a202-5095c82e760d.png)

**Comment by Zerogaku** [comment]
Running it with the `--defaultbg` or `-D` parameter works for me
